save-common/src/commonMain/kotlin/org/cqfn/save/core/plugin/PluginConfig.kt

```diff
@@ -86,7 +90,7 @@ data class GeneralConfig(
             this.suiteName ?: other.suiteName,
             this.excludedTests ?: other.excludedTests,
             this.includedTests ?: other.includedTests,
-            this.ignoreSaveComments ?: other.ignoreSaveComments
+            this.expectedWarningsPattern ?: other.expectedWarningsPattern,
```

Comment:
What about other plugins we have planned, like compile-and-run? Warnings pattern doesn't make much sense for them

Reply:
user is able not to provide it

Right now we should focus more on static analysis and it will be strange to have such duplication for the same file that contains both [fix] and [warn]